content/channels/index.textile

```diff
@@ -61,6 +61,7 @@ Namespaces can be used to apply operations to all channels within the namespace,
 <li>Exclude whitespace and wildcards, such as @*@</li>
 <li>Use the correct case, whether it be uppercase or lowercase</li></ul>
 <p>While Ably doesn't limit channel name and namespace length, be aware that the name appears in REST request URLs. Most browsers cap URLs at 2048 characters</p>
+<p>Namespaces don't need to be defined for a set of channels to be referred to in a capability. A resource specifier of foo:* (a glob expression) matches channel name foo:bar, even without foo namespace.</p>
```

Comment:
Thoughts on whether this should live outside of the note and as part of the text since the other items are referring to restrictions?

Reply:
I agree -- Let's move it

Reply:
@m-hulbert, your feedback has been addressed here: fd46e72c97f1acd3a2a494b5577661b9ae18c08d